package.json

```diff
@@ -1,24 +1,32 @@
 {
   "name": "warzone",
   "version": "1.0.0",
-  "description": "A warzone website stats",
+  "description": "A Call of Duty: Warzone stats tracking site for gamers and their squads",
   "main": "index.js",
+  "exact-engines": {
+    "node": "12.20.0",
+    "npm": "6.14.8"
+  },
   "scripts": {
     "test": "mocha --require esm",
     "build": "next build",
-    "nodemon": "nodemon",
+    "nodemon": "npm run pre-start && nodemon",
     "reminder": "node config/dev/reminders.js",
     "refresh-materialized-views": "ts-node lib/etl/jobs/materialized_views_refresh.ts",
     "refresh-gamer-matches": "ts-node lib/etl/jobs/user_match_runner.ts --jobType refresh",
     "backfill-gamer-matches": "ts-node lib/etl/jobs/user_match_runner.ts --jobType backfill",
     "update-gamer-matches": "ts-node lib/etl/jobs/user_match_runner.ts --jobType update",
     "view-logs": "heroku logs -t",
-    "dev": "ts-node server.ts",
+    "dev": "npm run pre-start && ts-node server.ts",
+    "dev-start-only": "ts-node server.ts",
+    "pre-start": "ts-node config/dev/preStart.ts",
     "_dev": "next dev",
     "start": "NODE_ENV=production ts-node server.ts",
     "etl": "ts-node src/lib/etl/user_match_scrape.js",
+    "git-add": "git add .",
```

Comment:
This was added so we can eventually run lint-fix then git-add to quick fix and quick git add before a commit, throwing the error if something bigger is broken